docs/design.md

```diff
@@ -112,12 +112,12 @@ The following scheme can be used for storing artifacts for Models, Datasets, and
 
 ```sh
 # Models
-gs://{bucket}/{hash}/model # Model artifacts (*.pt, etc)
-gs://{bucket}/{hash}/logs  # Logs and converted notebooks
+gs://{bucket}/{hash}/model     # Model artifacts (*.pt, etc)
```

Comment:
maybe we could standardize on `gs://{bucket}/{hash}/data` for both models and datasets? Both datasets and models can store arbitrary files in arbitrary formats. In the end it's storing the Model's data and storing the Datasets'  data.

I don't have a preference, just bringing it up in case that makes stuff easier.

Reply:
Feel free to resolve without responding if you think nah.

Reply:
I went back-n-forth on a similar take: each object using `{hash}/artifacts`. I decided against that b/c we might want to store different types of "data" or "artifacts" and I thought it made sense to split those things by their types in directories.